DOCS/man/options.rst

```diff
@@ -2639,6 +2639,10 @@ Subtitles
         particular subtitles in mkv files) are always assumed to be UTF-8.
 
 
+``--sub-stretch-durations=<yes|no>``
+    Stretch a subtitle duration so it ends when the next one starts.
+    Should help with subtitles which erroneously have zero durations.
```

Comment:
Sorry just thought of this later, but could you clarify that this option only has an effect for text subtitles?

Reply:
Done (I think).

Reply:
LGTM.